R/oxcAARCalibratedDate.R

```diff
@@ -81,13 +81,138 @@ format.oxcAARCalibratedDate <- function(x, ...){
 print.oxcAARCalibratedDate <- function(x, ...) cat(format(x, ...), "\n")
 
 ##' @export
-plot.oxcAARCalibratedDate <- function(x, ...){
-  #   if (requireNamespace("ggplot2", quietly = TRUE)) {
-  #     plotoxcAARDateGGPlot2(x, ...)
-  #   } else {
-  plotoxcAARDateSystemGraphics(x, ...)
-  # }
+plot.oxcAARCalibratedDate <- function(x, use_ggplot=T, ...){
+  if (requireNamespace("ggplot2", quietly = TRUE) & use_ggplot) {
+    plotoxcAARDateGGPlot2(x, ...)
+  } else {
+    plotoxcAARDateSystemGraphics(x, ...)
+  }
 }
+
+#' @importFrom "ggplot2" "ggplot" "aes" "geom_area" "labs" "theme_light" "geom_ribbon" "scale_y_continuous" "sec_axis" "ggplot_build" "geom_polygon" "scale_x_continuous" "annotate" "geom_errorbarh" "scale_alpha_manual"
+#' @importFrom "stats" "dnorm"
+plotoxcAARDateGGPlot2<-function(x, ...){
+  bc <- .data <- NULL
+  to_plot <- data.frame(dates=x$raw_probabilities$dates,
```

Comment:
Horrible code layout, but you do you. 

I don't see anything wrong with the implemented logic :+1: